scenarios4/02_Into_Our_Frozen_Land.cfg

```diff
@@ -71,7 +75,8 @@
         canrecruit=yes
         recruit=Vampire Guard,Vampire Master,Vampire Master Bowman
         side=2
-        {GOLD 75 100 125}
+        {GOLD 75 100 150}
+        {INCOME 0 0 12}
```

Comment:
Add some income on normal as well, please. Other places as well.

Reply:
Ok, but I don't see the point.  In most cases, the additional income is needed to offset the cost of the AI recruiting/supporting higher level units.  In others, to avoid the classic 'fight off one big wave on favorable terrain and then advance and mop up'.  Since I'm generally not changing NORMAL and it was fine before with income=0 it should be fine no matter what changes on HARD.

Reply:
But I think that Normal could benefit from some of these changes to some extent.